se_feedback/product-vision/feedback-product-vision.md

```diff
@@ -6,6 +6,18 @@
 | Reviewer   | Aaron Ang             |
 | Date       | 14 May 2016           |
 
+## Grade
+
+Grade: **8.6**
+
+| Questions                                                                                                                                         | Grade |
+|---------------------------------------------------------------------------------------------------------------------------------------------------|-------|
+| Who is going to buy the product? Who is the target customer?                                                                                      | 10    |
+| Which customer needs will the product address?                                                                                                    | 7     |
+| Which product attributes are crucial to satisfy the selected needs, and therefore to the success of the product?                                  | 8     |
+| How does the product compare against existing products, both from competitors and the same company? What are the product’s unique selling points? | 8     |
```

Comment:
Since there is no feedback in the "Product Uniqueness" section, how come we got an 8 for this part? :)